Project/Assets/ML-Agents/Examples/SharedAssets/Scripts/ModelOverrider.cs

```diff
@@ -25,7 +25,6 @@ namespace Unity.MLAgentsExamples
     public class ModelOverrider : MonoBehaviour
     {
         HashSet<string> k_SupportedExtensions = new HashSet<string> { "nn", "onnx" };
-        const string k_CommandLineModelOverrideFlag = "--mlagents-override-model";
```

Comment:
This was an obsolete way of specifying the override; we always use `mlagents-override-model-directory` now.